webapp/apps/taxbrain/helpers.py

```diff
@@ -162,8 +165,8 @@ def string_to_float_array(s):
     'df_dec': 'Difference between Base and User plans by expanded income decile',
     'mX_bin': 'Base plan tax vars, weighted avg per expanded income bin',
     'mY_bin': 'User plan tax vars, weighted avg per expanded income bin',
-    'df_bin': 'Difference between Base and User plans by expanded income bin',
-    'fiscal_tots': 'Total Revenue Change by Calendar Year',
+    'df_bin': 'Individual Income Tax: Difference between Base and User plans by AGI decile',
```

Comment:
Yea, haven't been able to double check with @MattHJensen but I think you're right. Will change this in a sec.